docs/general/resources/res_desc.json

```diff
@@ -3233,5 +3233,49 @@
         "desc": "Unused",
         "defaults": {},
         "note": ""
+    },
+    "502": {
+        "name": "Chopping food productivity",
+        "desc": "Chopping food generation rate per 100 seconds",
+        "defaults": {
+            "0": " Default",
+            "4": " Athenians"
+        },
+        "note": ""
+    },
+    "503": {
+        "name": "Trade wood percentage",
+        "desc": "Amount of trade will be returned as wood instead of gold",
+        "defaults": {
+            "0": " Default"
+        },
+        "note": "Setting it to 0.5 will return half the amount in gold and half in wood. Seems to be working for trade cogs only. Setting it to a value >= 1 will make it so no trade resources are returned"
+    },
+    "507": {
+        "name": "Unknown Athenian resource",
+        "desc": "Unknown... What does this resource do?",
+        "defaults": {
+            "0": " Default",
+            "1": " Athenians"
+        },
+        "note": "Unknown, but probably something related to Athenian policies"
+    },
+    "521": {
```

Comment:
Are resources 508-520 unused? If so, could you copy the format from one of the unused resources above and create "Unused Resource 509" to "Unused Resource 520" for completeness? If their purposes are unknown, then use "Unknown" instead of "Unused" in the name

Reply:
since the dlc patch i think they increased the resource limit by 100. 
and these are the resources i found in a.g.e. effects.